app/views/admin/events/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>
+  Event list
+  <a href="events/new" class="btn btn-primary active" role="button" style="float: right;">New</a>
```

Comment:
here you should use the helper `link_to`
Like:
`<%= link_to "Events", admin_events_path %>`
Probably the path will be `new_admin_events_path`. You can run `docker-compose run web bin/rake routes` to check all available routes, and add a `_path` at the end.